src/method/technology.tex

```diff
@@ -0,0 +1,94 @@
+\section{Technology}\label{sec:technology}
+
+Throughout the development of the project, a number of different tools and publicly available packages are used to
+assist the development process.
+These different technologies were chosen based on the team's needs and the requirements of the project.
+The following sections will describe the different technologies used in the project.
+
+\subsection{Programming stack}\label{subsec:programming}
+
+As the project is a web application, TypeScript~\cite{typescript} is used as the main programming language.
+TypeScript is a superset of JavaScript, with the main difference being that TypeScript is statically typed, while
+JavaScript is dynamically typed.
+It can be very useful in larger projects, as it helps to catch errors during compile-time as opposed to run-time.
+This is the first time for many of the team members to use TypeScript, but they find it to be safer than JavaScript
+for larger projects.
+
+The frontend of the project is built with React~\cite{react}, which is a JavaScript / TypeScript library for building
+user interfaces.
+It is developed and maintained by Meta, and is used by many large companies.
+React is focused on mobile applications first and foremost, but the team's focus was on the desktop experience.
+
+% textidote: ignore begin % due to node.js and next.js being identified as an url
+The backend is built using Node.js~\cite{node.js}, which allows the team to build server-side components using
+TypeScript.
+Node.js is a popular choice for backend development, as it is fast and scalable.
+As well as using React for client-side rendering, Next.js~\cite{next.js} is incorporated to handle the server-side
+rendering.
+Next.js is a React framework running on top of Node.js, which which assists in building the frontend and backend
```

Comment:
'Which' two times

Reply:
I blame this on textidote (which I ignore in this case)